<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Deep Sequencing for Respiratory Pathogen Detection &amp; Infection Diagnosis</title>
    <meta charset="utf-8" />
    <meta name="author" content="Brendan J. Kelly, MD, MS" />
    <meta name="date" content="2021-11-05" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <script src="libs/xaringanExtra-webcam/webcam.js"></script>
    <script id="xaringanExtra-webcam-options" type="application/json">{"width":"200","height":"200","margin":"1em"}</script>
    <link href="libs/tachyons/tachyons.min.css" rel="stylesheet" />
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <script src="libs/clipboard/clipboard.min.js"></script>
    <link href="libs/shareon/shareon.min.css" rel="stylesheet" />
    <script src="libs/shareon/shareon.min.js"></script>
    <link href="libs/xaringanExtra-shareagain/shareagain.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-shareagain/shareagain.js"></script>
    <link rel="stylesheet" href="xaringan-themer-inferno-grids-fade-title.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">

exclude: true





---
class: top left hide-count
background-image: url(img/PXL_20211017_215933273_trim.jpeg)
background-size: cover

.fade-in[

.move-top10[
.title-text[Deep Sequencing for&lt;br&gt; Respiratory Pathogen Detection&lt;br&gt; &amp; Infection Diagnosis]  
]  
  
.callout-url-leftcorner[
.title-nametext[
Brendan J. Kelly, MD, MS  
Infectious Diseases &amp; Epidemiology  
University of Pennsylvania  
&lt;br&gt;
&lt;br&gt;
&lt;!--- 05 November 2021  ---&gt;
]
]
]


---

## Disclosures  

.pad-left[

- No commercial support or conflicts of interest.    

- Research supported by:  
  
    - NIAID: K23 AI121485    
    
    - CDC: BAA 200-2016-91964, BAA 200-2018-02919, BAA 200-2021-10986   
  
]


---

## Respiratory Tract Infection: Public Health Challenge       
  
.pad-left[  
  
- Lower respiratory tract infections (LRTIs) were the fourth leading cause of mortality for all ages in 2019 -- even before the global COVID-19 pandemic!    
  
- LRTIs account for nearly 4 million deaths annually.  
  
- Multidrug-resistant (MDR) bacterial LRTIs have been associated with increased reinfection risk and mortality, relative to non-resistant bacterial infections:  
  
    - effect of factors that increase risk for MDR (e.g., underlying lung disease)?  
    
    - effect on time to effective antimicrobial therapy?  
    
      ... is microbiologic diagnosis complete without resistance profile?   
  
]  
  
  
.footnote[Vos et al. _Lancet_ 2020; Vincent et al. _JAMA_ 2009; Weiner et al. _MMWR_ 2016; Kumar et al. _Crit Care Med_ 2006]  
  
  

---

## Respiratory Tract Infection: Diagnostic Challenge       
  
.pad-left[  
  
- Challenges to sensitivity:  

    - potential etiologies include viruses, bacteria, fungi  
    
    - low microbial biomass and/or fastidious organisms  
    
    - difficulty of sampling (e.g., nonproductive cough)  
  
- Challenges to specificity:  
  
    - all non-surgical samples contaminated by the upper airway  
    
      (bronchoscopic carryover even with protected-specimen brush)  

  
]  
  
.footnote[Charlson et al. _Am J Resp Crit Care_ 2011; Charlson et al. _PLoS One_ 2012; Dickson et al. _Ann Rev Physio_ 2016]  
  
  
  
---

## Deep Sequencing for Culture-Free Diagnosis           
  
.pad-left[

- Approach:  
  
    - extraction of nucleic acid from upper/lower respiratory specimen →   
      16S rRNA gene or shotgun metagenomic sequencing   
  
- Potential advantages:      

    - rapid (depending on bioinformatics) &amp; less biased (but not unbiased)        
    
- Potential disadvantages:  

    - low sensitivity: microbial DNA swamped by host   
    
    - misclassification risk: metagenomic assignment/assembly  
    
    - limited accuracy of genotype-based AMR prediction  


]


  
  
  
---
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-a animated flipInY"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;





---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-a"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-b animated flipInX"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;





---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-a"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-b"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-c animated bounceInDown"&gt;&lt;div&gt;Dis/Advantages&lt;br&gt;of a Sequence-&lt;br&gt;First Strategy&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;





---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-a"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Dis/Advantages&lt;br&gt;of a Sequence-&lt;br&gt;First Strategy&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;





---
exclude: true
class: center middle



&lt;iframe src="./figs/ggplot_plotly_raster_demo.html" width = "95%", height = "95%" framBorder="0"&gt;&lt;/iframe&gt;



---
background-image: url(img/kelly_microbiome_2016_heatmap.png)
background-size: 70%




.footnote[Kelly et al. _Microbiome_ 2016]   



---
  
## 16s rRNA Sequencing for Bacterial Pneumonia    
  
.pad-left[

- concordance with clinical cultures:  

    - 4/15 subjects with + LRT cultures  
    
    - 16S rRNA identifies all as dominant 24-72hrs before culture  

- dominance of unknown clinical significance:  

     - _Enterococcus faecalis_ domination post RSV pneumonia  
     
     - _Ureaplasma parvum_ in an immunocompromised host  

- the "normal flora" problem: standards for ignoring expected taxa?  

]

.footnote[Kelly et al. _Microbiome_ 2016]   


---
background-image: url(img/pendleton_ajrccm_2017_top.png)
background-size: contain



.footnote[Pendleton et al. _AJRCCM_ 2017]   


---
background-image: url(img/pendleton_ajrccm_2017_bottom.png)
background-size: contain



.footnote[Pendleton et al. _AJRCCM_ 2017]   


---
  
## 16s rRNA &amp; Metagenomic Pathogen Detection      
  
.pad-left[

- high sensitivity &amp; early detection of culture-positive cases    

- require tailored methods for _Mycobacteria_, fungi, viruses     

- lots of dominance of unclear clinical significance:  
  
    - correlates with markers of inflammation and lung injury  
    
    - should this receive directed antibiotic treatment?  
    
    - byproduct of antibiotic treatment?  
  
]

.footnote[Graf et al _JCM_ 2016; Abbas _Am J Transpl_ 2017; Kitsios et al. _Front Micro_ 2018; Langelier et al. _PNAS_ 2018; Zinter et al. _CID_ 2019; Dickson et al. _AJRCCM_ 2020; McGinniss et al. _J Heart Lung Transpl_ 2021]   



---

## Conclusions: Sequence-Based Pathogen Detection              
  
.pad-left[
  
- Advantages:      

    - rapid &amp; less biased        
    
- Disadvantages:  

    - need new standard for "normal flora"  
  
    - no single extraction/sequencing modality: _Mycobacteria_, fungi, viruses  
    
      (efforts made to develop pipelines -- e.g., SURPI*)  
    
    - AMR prediction in its infancy  
    
- Best of both worlds with multiplex NAAT?    
    
]

.footnote[Kelly et al. _Microbiome_ 2016; Kitsios et al. _Front Micro_ 2018; *Langelier et al. _PNAS_ 2018; Dickson et al. _AJRCCM_ 2020]   
  
  



---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-b"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Dis/Advantages&lt;br&gt;of a Sequence-&lt;br&gt;First Strategy&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;





---
background-image: url(img/kelly_microbiome_2016_alpha_simple.png)
background-size: 70%


.footnote[Kelly et al. _Microbiome_ 2016]   



---
  
## Dominance, Diversity &amp; Infection Risk        
  
.pad-left[

- more comprehensive and quantitative approach permits:  

    - &lt;u&gt;richness&lt;/u&gt;: number of different species  
    
    - &lt;u&gt;evenness&lt;/u&gt;: balance across observed species  
    
    - &lt;u&gt;diversity&lt;/u&gt;: summary of richness &amp; evenness  
    
    - &lt;u&gt;dominance&lt;/u&gt;: &gt; 30% or &gt; 50% of sequence reads  
    
- 16S rRNA qPCR or other standard-based quantitation of DNA/RNA abundance    
  
  (use of cycle threshold values for SARS-CoV-2)  
  
]

.footnote[Dickson et al. _Lancet Resp Med_ 2014; Kelly et al. _Microbiome_ 2016; Kitsios et al. _Front Micro_ 2018; Dickson et al. _AJRCCM_ 2020; Kitsios et al. _AJRCCM_ 2020; Harrigan et al. _CID_ 2021]   


---
background-image: url(figs/p_combined_mdi_crosssection_counterfact.png)
background-size: contain



.footnote[Harrigan et al. _CID_ 2021]   

---
background-image: url(figs/p_minimpute_persistent_mdi.png)
background-size: contain



.footnote[Harrigan et al. _CID_ 2021]   

---
background-image: url(figs/p_minimpute_compare_outcome_definitions_persistence.png)
background-size: contain



.footnote[Harrigan et al. _CID_ 2021]   

---
  
## Dominance, Diversity &amp; Infection Risk        
  
.pad-left[
  
- 83 subjects dependent on mechanical ventilation  

- Sampled 1066 days to compare microbiome disruption indices (MDIs):  
  
    - cross-sectional MDIs (low Shannon diversity, high 16S rRNA gene qPCR) associated with risk for VA-LRTI, but poor predictors  

    - persistent low Shannon diversity had per-day LRTI odds ratio of 1.36 (95% credible interval 1.10 to 1.72)

    - observed association was consistent across multiple definitions of VA-LRTI  
  
  
]

.footnote[Harrigan et al. _CID_ 2021]   


---

## Conclusions: Microbial Ecology of Infection    
  
.pad-left[

- Predictive utility of microbial community structure:    

    - challenge of gold standard for pneumonia diagnosis
    
    - optimal utility with longitudinal measures/sampling  
   
    - independent of pathogen identification (strength &amp; weakness)  
    
- Advantage of deep sequencing over multiplex NAAT.  
  
- Is it worth the work?
  
  
]


---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-grey"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-c"&gt;&lt;div&gt;Dis/Advantages&lt;br&gt;of a Sequence-&lt;br&gt;First Strategy&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;






---

## Deep Sequencing First?    
  
.pad-left[

- Disadvantages:  

    - need to redefine "normal flora" (specificity questions) 
    
    - broad sensitivity demands complex wet-bench &amp; bioinformatic pipeline  
    
    - metagenomic AMR prediction is in early stages  

- Advantages:  

    - speed and sensitivity  
    
    - added predictive/diagnostic value of microbial community structure  
    
    - __pathogen tracking and infection control__ -- application to public health   

]




---
background-image: url(figs/covid_cases_chloropleth.png)
background-size: 30%
background-position: 85% 50%

## SARS-CoV-2 Variant Tracking     
  
.pull-left[

- The value of extracted nucleic acids!  
  
- Draw on broad catchment of PennMedicine, led by Kyle Rodino, PhD:    
  
    - ~ 1500 SARS-CoV-2 NAATs per day
    
    - positives with Ct &lt; 28, stratified by county and zip code   
    
    - ARTIC primer set to amplify → Illumina    
    
    - assembly, alignment, PANGO lineage    

]

.pull-right[




]



---
background-image: url(img/marques_arma_model.png)
background-size: 80%
background-position: 50% 90%

.move-top5[
&lt;img src="./img/marques_arma_model_key.png" width="70%" style="display: block; margin: auto;" /&gt;
]

.footnote[Marques et al. _medRxiv_ 2021]   


---
background-image: url(img/marques_vaccine_model.png)
background-size: contain


.footnote[Marques et al. _medRxiv_ 2021]   


---
layout: false
class: full-screen hide-count
background-color: #FCF5E5


&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile gridset-a animated flipInX"&gt;&lt;div&gt;Sequence-Based&lt;br&gt;Pathogen&lt;br&gt;Detection&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-b animated flipInY"&gt;&lt;div&gt;Microbial&lt;br&gt;Ecology &amp;&lt;br&gt;Infection Diagnosis&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile gridset-c animated flipInX"&gt;&lt;div&gt;Dis/Advantages&lt;br&gt;of a Sequence-&lt;br&gt;First Strategy&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;


---

## Conclusions  

.pad-left[

- Specificity challenges deep sequencing for respiratory pathogen identification:  

    - nature of upper airway contamination  
    
    - need to redefine standards for "normal flora"
  
- Novel insights into microbial ecology provide alternate framework for infectious disease diagnosis: skip the CT scan and monitor lower respiratory bacterial community diversity/dominance?  
  
- As multiplex NAAT and diagnostic sequencing applications expand, opportunity to collaborate with infection control and public health colleagues to build bioinformatics infrastructure.  

]


---

## Acknowledgements    

.pull-left[
- __ARES Group @ Penn__
Laurel Glaser, Kyle Rodino, Sean Loughrey, Laura Cowden,   
Magda Wernovsky, Jennifer Han, Erik Clarke, Michael David,
Matt Ziegler, Lauren Dutcher, Ebbing Lautenbach, Jim Harrigan, Hatem Abdallah

- __Bushman Laboratory @ Penn__
Arwa Abbas, Aoife Roche, Andrew Marques,
Aubrey Bailey, Jacob Leiby,
Rick Bushman

- __PennCHOP Microbiome Program__
Lisa Mattei, Casey Hofstaedter, Huanjia Zhang, Kyle Bittinger

]


.pull-right[
- __Collman Laboratory @ Penn__
Ize Imai, Aurea Simon Soro, John McGinniss, Ron Collman

- __Division of ID @ Penn__
Ian Frank, Pablo Tebas, Robert Gross, Emily Blumberg

- __Rutgers University &amp; Penn DBEI__
Jason Roy, Arman Oganisian
  
- __CDC Prevention Epicenters__
Clifford McDonald, Alison Laufer Halpin
  
- __Funding__
CDC BAAs 200-2016-91964, 200-2018-02919, 200-2021-10986;
NIAID: K23 AI121485

]

.center[
### brendank@pennmedicine.upenn.edu
]



---
class: middle center hide-count
background-image: url(img/PXL_20211017_215933273_trim.jpeg)
background-size: cover

.title-subtext[Questions?]

.callout-url-bottom[  
 &lt;!-- .fade-in[&lt;span style="font-size:0.5em; color:#113864"&gt;slides ↓ &lt;/span&gt;] --&gt; 
&lt;span style="color:black; link-color:black;"&gt;[bjklab.org](http://www.bjklab.org)&lt;/span&gt;  
&lt;br&gt;  
]  


---
exclude: true

## bjklab.org
  
.center[
&lt;iframe src="http://bjklab.org" width="95%" height="400px" data-external="1"&gt;&lt;/iframe&gt;
]


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"slideNumberFormat": "%current%"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
